### PR TITLE
Got rid of remote option for proxy

### PR DESCRIPTION
**Pull request by rwmtse**
Don't merge yet
@bryanhiggins Please ask Jenny to help testing this.

### CODE REVIEW

File: lib/client/constants.js [hunk 1]
```diff
@@ -206,8 +204,7 @@ module.exports = {
     "XHR": {
         PROXY_SETTING: "settings-xhr-proxy-setting",
         PROXY_SETTINGS_LIST: {
-            remote: "remote",
-            local: "local",
+            enabled: "enabled",
```
Comment (bryanhiggins) [inline lib/client/constants.js:207]: Where is this checked?

**Comment by bryanhiggins** [comment]
@rwmtse I don't see any change to settings.js, which is currently defaulting to "local". I suggest you grep for PROXY_SETTINGS_LIST.

We also want an error printed to console if XHR is attempted with proxy enabled, but ripple services have not been started.

**Comment by rwmtse** [comment]
@bryanhiggins I've changed settings.js, as to detecting if the service is started, I am not sure if know how to do it, I spent 2 hours looking at the code, and my understanding is that client side files should not require server side files (all client-side require is done with "ripple()" which only require files under client, so... this means in order to check the service is started, I'd have to test it with an XHR? Is that what you want me to do? Not sure if I'm missing something...

**Comment by bryanhiggins** [comment]
@rwmtse Just print out a message anytime the onerror handler is invoked.

"Error handing proxy request. Please ensure ripple services have been started."

**Comment by rwmtse** [comment]
@bryanhiggins I attached an error listener to the xhr and print the message there, see cors.js and jsonp.js

**Comment by bryanhiggins** [comment]
r+

**Comment by jengee** [comment]
tested this on Mac.
By default, cross domain proxy is enabled by default.
correct errors show regarding to cross origin requests are made when ripple does/does not start service as well as enabling enable/disable cross domain proxy
Pass.

**Pull request closed without merging**